src/test/java/plugins/LogParserTest.java

```diff
@@ -0,0 +1,97 @@
+package plugins;
+
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.plugins.logparser.LogParserGlobalConfig;
+import org.jenkinsci.test.acceptance.plugins.logparser.LogParserOutputPage;
+import org.jenkinsci.test.acceptance.plugins.logparser.LogParserPublisher;
+import org.jenkinsci.test.acceptance.po.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.jenkinsci.test.acceptance.junit.Resource;
+import org.openqa.selenium.WebElement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.jenkinsci.test.acceptance.Matchers.*;
+
+@WithPlugins("log-parser")
+public class LogParserTest extends AbstractJUnitTest {
+
+    // GlobalConfig for the LogParser-Plugin
+    private LogParserGlobalConfig config;
+    // Available Rules for the tests
+    private Map<String, String> rules;
+
+    @Before
```

Comment:
Within the @Before function multiple non-static references, like jenkins, are accessed. When is a new instance of jenkins created? On each start of a test case or test class?